pkg/term/term_windows.go

```diff
@@ -1,12 +1,15 @@
 // +build windows
+
 package term
 
 import (
+	"fmt"
 	"io"
 	"os"
+	"os/signal"
 
-	"github.com/Sirupsen/logrus"
-	"github.com/docker/docker/pkg/term/winconsole"
+	// "github.com/Sirupsen/logrus"
```

Comment:
@brendandixon Ah my bad, I mixed up log outputs and user feedback. Never mind then.